code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -15,16 +16,18 @@
 /obj/item/weapon/reagent_containers/glass/bottle/New()
 	..()
 	if(!icon_state)
-		icon_state = "bottle[rand(1,20)]"
+		icon_state = pick("reagent_bottle","rounded_bottle")
```

Comment:
reagent bottles are distinct from regular bottles, especially since they contain nearly double the volume--probably shouldn't be setting this as a potential option.